### PR TITLE
Add Reuven Lax to committers list.

### PR DESCRIPTION
**Pull request by reuvenlax**

### CODE REVIEW

File: src/_beam_team/team.md [hunk 1]
```diff
@@ -92,6 +92,11 @@ members:
     organization: data Artisans
     roles: committer, PMC
     time_zone: "+1"
+  - name: Reuven Lax
+    apache_id: reuvenlax
+    email: reuvenlax [at] apache [dot] org
+    organization: Google
+    roles: committer
```
Comment (davorbonaci) [inline src/_beam_team/team.md]: timezone?

Comment (reuvenlax) [inline src/_beam_team/team.md]: Done

**Comment by tgroh** [comment]
@asfgit merge

**Comment by asfgit** [comment]
Error: PR failed in verification; check the Jenkins job for more information.

**Comment by kennknowles** [comment]
retest this please

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/634/

Jenkins built the site at commit id fd26dc8c33a6949dcb7ee51eb82986c217f4bf8a with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/288/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Pull request closed without merging**